package.json

```diff
@@ -12,6 +12,7 @@
       "**/react",
       "**/@react-navigation/**",
       "**/react/**",
+      "**/lodash**",
```

Comment:
What was this change required for?

Reply:
`lodash.isequal` is a new dependency of the `react-native-reanimated` package, we need to keep it in the sample app directory to make Metro bundle it at runtime.